src/containers/Layout/Layout.js

```diff
@@ -7,12 +7,32 @@ import Sidebar from "../../components/Sidebar/Sidebar";
 import Jokes from "../Jokes/Jokes";
 
 class Layout extends Component {
+  constructor(props){
+    super(props);
+
+    this.state = {
+      jokesList: [],
+    }
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event) {
+    const target = event.target;
+    const name = target.name;
+
+    fetch( 'https://api.chucknorris.io/jokes/random?category=' + name )
+      .then( response => response.json() )
+      .then( data => this.setState({jokesList: [data, ...this.state.jokesList]}) );
+  }
+
   render() {
+    console.log(this.state.jokesList);
```

Comment:
Its a good practice to remove all `console.log` statements from the code after feature is finished and ready for merge. If you really need it there are some advanced webpack options what remove all console.log statements during the build process